action.yml

```diff
@@ -22,8 +22,3 @@ inputs:
 runs:
   using: 'docker'
   image: 'Dockerfile'
-  args:
```

Comment:
why was this removed?

Reply:
it is not used, all inputs are set as environment variable.
you don't use nowhere variable like $1, $2, ...

Reply:
just read up on them, looks good